content/20.treatments.md

```diff
@@ -72,25 +72,35 @@ Differently from a DNA virus, which can use the host enzymes to propagate itself
+Favipiravir (6-fluoro-3-hydroxy-2-pyrazinecarboxamide) acts as a purine and purine nucleoside analogue that inhibits viral RNA polymerase in a dose dependent manner across a range of RNA viruses, including Influenza virus [@doi:10.1128/AAC.49.3.981-986.2005; @doi:10.1128/AAC.01074-08; @doi:10.1128/AAC.00356-07; @doi:https://doi.org/10.1016/j.bbrc.2012.07.034; @doi:10.1128/AAC.01219-10]. 
+Evidence for T-705 inhibiting viral RNA polymerase are based on time-of-drug addition studies that found that viral loads were reduced with the addition of Favipiravir in early times post-infection [@doi:10.1128/AAC.49.3.981-986.2005; @doi:10.1016/j.bbrc.2012.07.034; @doi:10.1128/AAC.01219-10].
+Additionally, biochemical experiments showed that Favipiravir was recognized as a purine nucleoside analogue and incorporated into the viral RNA template. 
+A single incorporation does not influence RNA transcription, however multiple events of incorporation lead to the arrest of RNA synthesis [@doi:10.1371/journal.pone.0068347].
 
 ##### Current Evidence
-
-A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
+The effectiveness of Favipiravir for treating patients with COVID-19 is currently under investigation.
+An open-label, nonrandomized, before-after controlled study was recently conducted [@doi:10.1016/j.eng.2020.03.007].
+The study included 80 COVID-19 patients (35 treated with Favipiravir, 45 control) from the isolation ward of the national clinical research center for infectious diseases (The Third People’s Hospital of Shenzhen), Shenzhen, China.
+The patients in the control group were treated with the other anti-virals, such as Lopinavir and Ritonavir.
+Treatment applied on days 2-14; treatment stopped when viral clearence was confirmed or at day 14.
+The efficacy of the treatment was measured by 1)the time until viral clearance using Kaplan-Meier survival curves and 2)the improvement rate of chest computed tomography (CT) scans on day 14 after treatment.
+The study found that Favipiravir increased the speed of recovery (measured as viral clearance from the patient by RT-PCR) to 4 days compared to 11 days using other antivirals such as lopinavir and ritonavir. 
+Additionally, the lung CT scans of patients treated with Favipiravir showed significant improvement rates (91%) on day 14 compared to the 62% of control patients.
+However, there were adverse side effects in 4 (11%) Favipiravir-treated patients and 25 (56%) control patients.
+The adverse side effects included: diarrhea, vomiting, nausea, rash, liver and kidney injury.
+Overall, despite the study reporting clinical improvement in Favipiravir-treated patients, due to the some issues with study design, it cannot be determined whether treatment with Favipiravir had an effect or whether these patients would have recovered regardless of any treatment.
+For example, the sample size is too small to give a powerful statistics.
```

Comment:
```suggestion
For example, the sample size is too small to adequately power the statistical analysis.
```
Did they do statistical analyses that you're critiquing? If they refrained from doing inappropriate stats, then the word "the" should be removed from my suggestion

Reply:
They performed chi-square tests to compare categorical data, and the Wilcoxon rank-sum test or Student’s t test for continuous data. I'm not sure what is an appropriate sample size needed, but this does seem like a reasonable number to get some significance but I wasn't sure as a non-stats person. This is some of the old text that I didn't update from the original author.

Reply:
Ah! Thanks Alex. This text seems to be referring to the Kaplan Meier survival curve analysis which is not something I'm particularly comfortable speculating about coming from a psych/genomics stats background! It seems like they should have done a power analysis as part of study design but I don't see any info in the manuscript. Honestly it doesn't look egregiously underpowered to me, the IQR are almost non-overlapping for the two treatment conditions.
Perhaps this hedges a bit better?:
```suggestion
For example, although there were significant differences between the two treatment groups, follow-up analysis is necessary due to the small sample size.
```

Reply:
I agree! I'm also not a super stats person, but I was also thinking that some sort of power analysis would make sense and there wasn't one from what I read either. This IQR look distinct to me too..I think this new wording is a good balance